cmd/vdev_id/vdev_id

```diff
@@ -186,7 +186,7 @@ sas_handler() {
 
 		# Get the raw scsi device name from multipath -l.  Strip off
 		# leading pipe symbols to make field numbering consistent.
-		DEV=`multipath -l $DM_NAME |
+		DEV=`multipath -ll $DM_NAME |
```

Comment:
I updated the comment to be consistent with the code. I don't think we need to document the difference between -l and -ll here as that is covered in the multipath man page.